x-pack/plugins/apm/public/components/app/service_overview/service_overview_transactions_table/get_columns.tsx

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiBasicTableColumn } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { ValuesType } from 'utility-types';
+import {
+  asMillisecondDuration,
+  asPercent,
+  asTransactionRate,
+} from '../../../../../common/utils/formatters';
+import { APIReturnType } from '../../../../services/rest/createCallApmApi';
+import { px, unit } from '../../../../style/variables';
+import { SparkPlot } from '../../../shared/charts/spark_plot';
+import { ImpactBar } from '../../../shared/ImpactBar';
+import { TransactionDetailLink } from '../../../shared/Links/apm/transaction_detail_link';
+import { TruncateWithTooltip } from '../../../shared/truncate_with_tooltip';
+import { TransactionGroupsOverview } from './';
+
+export type ServiceTransactionGroupItem = ValuesType<
+  TransactionGroupsOverview['transactionGroups']
+>;
+type TransactionGroupMetrics = APIReturnType<'GET /api/apm/services/{serviceName}/transactions/groups/agg_results'>;
+
+function getLatencyAggregationTypeLabel(latencyAggregationType?: string) {
+  switch (latencyAggregationType) {
+    case 'avg':
+      return i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnLatency.avg',
+        { defaultMessage: 'Latency (avg.)' }
+      );
+
+    case 'p95':
+      return i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnLatency.p95',
+        { defaultMessage: 'Latency (95th)' }
+      );
+
+    case 'p99':
+      return i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnLatency.p99',
+        { defaultMessage: 'Latency (99th)' }
+      );
+  }
+}
+
+export function getColumns({
+  serviceName,
+  latencyAggregationType,
+  transactionGroupsAggResults,
+}: {
+  serviceName: string;
+  latencyAggregationType?: string;
+  transactionGroupsAggResults?: TransactionGroupMetrics;
```

Comment:
Do we have better names for both `transactionGroupsAggResults` and `TransactionGroupMetrics`? From just reading this I don't really understand why they are different and what they are referring to.

Reply:
I'm open to suggestions.

Reply:
Alright, let's just call them transaction group primary statistics and transaction group comparison statistics, and the merged variant transaction group statistics. Unless @sqren has a better suggestion :).

Reply:
`statistics` it is then!